src/main/java/org/killbill/billing/plugin/adyen/client/payment/service/AdyenCheckoutApiClient.java

```diff
@@ -34,7 +33,9 @@ public AdyenCheckoutApiClient(final AdyenConfigProperties adyenConfigProperties,
             environment = Environment.LIVE;
         }
 
-        final Client client = new Client(adyenConfigProperties.getApiKey(countryCode), environment);
+        //default API key KEY_NOT_FOUND, log it
+        final String apiKey = adyenConfigProperties.getApiKey(countryCode);
+        final Client client = new Client(apiKey, environment);
```

Comment:
Can we initialize Client with Detokenizer URL ?
Please add the test for the same.

Reply:
Added the config for url prefix. But Adyen SDK ignores url prefix for TEST environment.